tests/unit/test_python_splitter.py

```diff
@@ -94,28 +86,34 @@ def python_int_dataset(test_specs):
 
 @pytest.fixture(scope="module")
 def python_float_dataset(test_specs):
-    # fix the random seed
     np.random.seed(test_specs["seed"])
 
     # generates the user/item affinity matrix. Ratings are in the interval [0, 5), with 0s denoting unrated items.
-    return np.random.random(
+    return (
+        np.random.random(
             size=(test_specs["number_of_users"], test_specs["number_of_items"])
-        ) * 5
+        )
+        * 5
+    )
 
 
 def test_split_pandas_data(pandas_dummy_timestamp):
     splits = split_pandas_data_with_ratios(pandas_dummy_timestamp, ratios=[0.5, 0.5])
     assert len(splits[0]) == 5
     assert len(splits[1]) == 5
 
-    splits = split_pandas_data_with_ratios(pandas_dummy_timestamp, ratios=[0.12, 0.36, 0.52])
+    splits = split_pandas_data_with_ratios(
+        pandas_dummy_timestamp, ratios=[0.12, 0.36, 0.52]
+    )
     shape = pandas_dummy_timestamp.shape[0]
     assert len(splits[0]) == round(shape * 0.12)
     assert len(splits[1]) == round(shape * 0.36)
     assert len(splits[2]) == round(shape * 0.52)
 
     with pytest.raises(ValueError):
-        splits = split_pandas_data_with_ratios(pandas_dummy_timestamp, ratios=[0.6, 0.2, 0.4])
+        splits = split_pandas_data_with_ratios(
+            pandas_dummy_timestamp, ratios=[0.6, 0.2, 0.4]
+        )
 
 
 def test_min_rating_filter(python_dataset):
```

Comment:
This function doesn't test min_rating_filter thoroughly as it relies on random-generated data of 1000 records for 4 users, end up almost no chances to have the users whose `min_rating<5`. Should provide a dataset which actually includes those users.

Reply:
@loomlike, do you have time today to take a look at this? if not, @yueguoguo, do you have time tomorrow during your day? if not, I'll take a look tomorrow in my morning :-)

Reply:
@miguelgfierro where is plan C? haha I can take a look at this today. Will push the changes to this branch.